openfga_sdk/telemetry/metrics.py

```diff
@@ -122,12 +122,27 @@ def requestDuration(
             configuration = TelemetryConfiguration()
 
         if (
-            isinstance(configuration, TelemetryMetricsConfiguration) is False
+            isinstance(configuration, TelemetryConfiguration) is False
             or configuration.metrics.histogram_request_duration.enabled is False
             or configuration.metrics.histogram_request_duration.attributes() == {}
         ):
             return self.histogram(TelemetryHistograms.request_duration)
 
+        if (
+            value is None
+            and TelemetryAttributes.http_client_request_duration.name in attributes
```

Comment:
Can we drop the server and client request durations as attributes?

Reply:
Can you clarify? They're opt-in attributes presently; I'd imagine we'd want to keep them for alternate histograms, counters, etc?